src/media/css/app-list.styl

```diff
@@ -0,0 +1,72 @@
+/**
+ * App listing styles.
+ */
+
+@import 'lib';
+
+.app-list {
+    position: absolute;
+    top: $app-preview-height;
+    bottom: 0;
+    left: 0;
+
+    width: 100%;
+    overflow: auto;
+
+    ul {
+        padding: 4em 3em 3em 4em;
+    }
+
+    li {
+        float: left;
+        margin: 0 1em 1em 0;
+
+        list-style: none;
+    }
+}
+
+.app-list-app {
+    position: relative;
+
+    width: 150px;
+    height: 150px;
+    outline: 0;
+    border-radius: 100%;
+    overflow: hidden;
+
+    color: $greyscale-light-grey;
+    background-color: $greyscale-light-grey;
+
+    backface-visibility: hidden;
+
+    transition: border-width .5s,
+                transform .5s;
+
+    &:focus {
+        z-index: 1;
+
+        border: 8px solid $greyscale-white;
+
+        transform: scale(1.5);
```

Comment:
You'll notice this produces a blurry result in 1.5x since we can't assume the app icons are vector images. The transition itself is also awkward since it seems to take a moment to "settle" between its animating state and once the animation is finished. I don't have a great solution for this but you may have to experiment with changing the width/height or increasing the border to give the illusion of the size increase.